compiler/rustc_middle/src/ty/print/pretty.rs

```diff
@@ -393,6 +393,9 @@ pub trait PrettyPrinter<'tcx>:
         match self.tcx().trimmed_def_paths(()).get(&def_id) {
             None => Ok((self, false)),
             Some(symbol) => {
+                if Ident::with_dummy_span(*symbol).is_raw_guess() {
```

Comment:
If you print `Ident::with_dummy_span(*symbol)` instead of just `symbol`, then it will add `r#` automatically, `Display` impl for `Ident` does that.

Reply:
thanks for your tip